composer.json

```diff
@@ -185,7 +185,6 @@
         "nikic/php-parser":"^4.7",
         "doctrine/inflector": "^1.4",
         "sylius/grid-bundle": "1.7.4",
-        "symfony/form": "4.4.9|4.4.10|4.4.11|4.4.12",
         "symfony/symfony": "3.4.7",
```

Comment:
@lchrusciel This can be removed as well, the min reqs say: `^4.4` so you can't reach `3.x` ones ;)

Reply:
I will do it in separate PR ;)

Reply:
https://github.com/Sylius/Sylius/pull/11802